src/core/fonts.js

```diff
@@ -1920,6 +1926,22 @@ class Font {
         // not enough data for coordinates
         return glyphProfile;
       }
+
+      if (hintsValid && instructionsLength > 0) {
+        const ttContext = {
+          functionsDefined: [],
+          functionsUsed: [],
+          functionsStackDeltas: [],
+          tooComplexToFollowFunctions: false,
+          hintsValid: true,
+        };
+        sanitizeTTProgram(
+          { data: glyf.subarray(instructionsStart, instructionsLength) },
+          ttContext
+        );
```

Comment:
I suppose the question is how much all of this "extra" validation affects general font parsing performance, since there can be a lot of glyphs in fonts?

Although this is probably the correct thing to do in general, to catch more cases of broken fonts.